src/cli/idm/idm-export.js

```diff
@@ -0,0 +1,109 @@
+import { Command, Option } from 'commander';
+import * as common from '../cmd_common.js';
+import { getTokens } from '../../api/AuthApi.js';
+import {
+  exportAllConfigEntities,
+  exportAllRawConfigEntities,
+  exportConfigEntity,
+} from '../../ops/IdmOps.js';
+import storage from '../../storage/SessionStorage.js';
+import { printMessage } from '../../ops/utils/Console.js';
+
+const program = new Command('frodo idm export');
+
+program
+  .description('Export IDM configuration objects.')
+  .helpOption('-h, --help', 'Help')
+  .showHelpAfterError()
+  .addArgument(common.hostArgumentM)
+  .addArgument(common.realmArgument)
+  .addArgument(common.userArgument)
+  .addArgument(common.passwordArgument)
+  .addOption(common.deploymentOption)
+  .addOption(common.insecureOption)
+  .addOption(
+    new Option(
+      '-i, --idm-name <idm-name>',
+      'Name of an object. If specified, -a and -A are ignored.'
+    )
+  )
+  .addOption(
+    new Option(
+      '-f, --file [file]',
+      'Name of the file to write the exported object to. Ignored with -A.'
+    )
+  )
+  .addOption(
+    new Option(
+      '-ef, --entities-file [entities-file]',
+      'Name of the entity file to write the exported object to. Ignored with -A.'
+    )
+  )
+  .addOption(
+    new Option(
+      '-env, --env-file [envfile]',
+      'Name of the env file to write the exported object to. Ignored with -A.'
+    )
+  )
+  .addOption(
+    new Option(
+      '-a, --all <directory> [file] [envFile]',
+      'Export all the objects in a realm to a single file. Ignored with -i.'
+    )
+  )
+  .addOption(
+    new Option(
+      '-A, --all-separate <directory>',
```

Comment:
I'd like to keep the -A flag the same as in all the other commands supporting it, as a flag, not a param with value. There is a request to support -D across all commands (#209).